Mypy error after project restructure: Module "microvenv._create" has no attribute "main"

The full error message: microvenv\__main__.py:1: error: Module "microvenv._create" has no attribute "main"  [attr-defined]

Steps to reproduce:

1. clone the project
2. run `nox -s lint`

microvenv\__main__.py:1: error: Module "microvenv._create" has no attribute "main"  [attr-defined]

Full output: 

```nox -s lint
❯ nox -s lint
nox > Running session lint
nox > Creating virtual environment (virtualenv) using python.exe in .nox\lint
nox > python -m pip install '.[lint]'
nox > ruff check .
nox > black --check .
All done! ✨ 🍰 ✨
13 files would be left unchanged.
nox > mypy .
microvenv\__main__.py:1: error: Module "microvenv._create" has no attribute "main"  [attr-defined]
Found 1 error in 1 file (checked 11 source files)
nox > Command mypy . failed with exit code 1
nox > Session lint failed.
```

Versions:
win11 and py312

--- a/.readthedocs.yml
+++ b/.readthedocs.yml
@@ -9,7 +9,7 @@ version: 2
 build:
   os: ubuntu-22.04
   tools:
-    python: "3"
+    python: "3.11"
 
 # Build documentation in the "docs/" directory with Sphinx
 sphinx:

--- a/microvenv/__main__.py
+++ b/microvenv/__main__.py
@@ -1,3 +1,9 @@
+import sys
+
+if sys.platform == "win32":
+    print("Microvenv's CLI is not supported on Windows", file=sys.stderr)
+    sys.exit(1)
+
 from ._create import main
 
 if __name__ == "__main__":

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -5,7 +5,7 @@ build-backend = "hatchling.build"
 
 [project]
 name = "microvenv"
-version = "2023.3.post1"
+version = "2023.4"
 description = "A minimal re-implementation of Python's venv module plus utilities"
 keywords = ["virtual environments", "venv"]
 readme = "README.md"

